### PR TITLE
Features/initialmap

### PR DESCRIPTION
**Pull request by jasonelliots**
# Description: Map rendering incident data. 

## Type of change
Please delete options that are not relevant.
- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [X] This change requires a documentation update

## Change Status
- [ ] Complete, tested, ready to review and merge
- [X] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

# How Has This Been Tested?
No tests 

# Checklist
- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] My code has been reviewed by at least one peer
- [X] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [X] There are no merge conflicts

### CODE REVIEW

File: .env.sample [hunk 1]
```diff
@@ -1,3 +1,4 @@
 REACT_APP_CLIENT_ID=example
 REACT_APP_OKTA_ISSUER_URI="https://example-882474.okta.com"
-REACT_APP_API_URI=http://localhost:8005
\ No newline at end of file
+REACT_APP_API_URI=http://localhost:8005
```
Comment (jlong5795) [inline .env.sample:3]: Has this been added to the correct .env file?

File: src/components/map/Map.js [hunk 2]
```diff
@@ -194,25 +194,23 @@ const Map = () => {
...
-      .get('backendurl')
-      .then(res => {
-        console.log(res);
-        // save response into state
-      })
-      .catch(err => {
-        console.log(err);
-      });
-  }, []);
+  // useEffect(() => {
```
Comment (jlong5795) [inline src/components/map/Map.js]: No commented out code in main.

**Review by jlong5795 — CHANGES_REQUESTED** [review]
Please remove commented out code before merging to main branch.

File: .env.sample (hunk 1 above)
Comment (jasonelliots) [reply .env.sample:3]: Yes, we've got all the variables we need in the correct .env file.

File: src/components/map/Map.js (hunk 2 above)
Comment (jasonelliots) [reply src/components/map/Map.js]: Good catch, removing now.

**Review by jlong5795 — APPROVED** [review]
Thank you for making the requested changes.

**Pull request merged**